### PR TITLE
Add base R to Avant_Docker

### PR DESCRIPTION
**Pull request by danilito19**

### CODE REVIEW

**Comment by danilito19** [comment]
@tmertens

**Comment by tmertens** [comment]
cc/ @taylorzr we should merge this into the updated image branch you are working on.

File: Avant_Dockerfile [hunk 1]
```diff
@@ -156,6 +156,10 @@ RUN circleci-install docker_compose
 ARG TARGET_UNPRIVILEGED_LXC
 RUN if [ "$TARGET_UNPRIVILEGED_LXC" = "true" ]; then circleci-install circleci_docker; fi
 
+# R base
+ADD circleci-provision-scripts/r.sh /opt/circleci-provision-scripts/r.sh
+RUN circleci-install install_r
```
Comment (taylorzr) [inline Avant_Dockerfile]: This would just be `RUN circleci-install r`

**Comment by taylorzr** [comment]
Copied in the changes since I changed Avant_Dockerfile to Dockerfile.

**Pull request closed without merging**